packages/theme/src/cli/utilities/asset-ignore.ts

```diff
@@ -83,9 +85,18 @@ function matchGlob(key: string, pattern: string) {
     noglobstar: true,
   }
 
-  const result = originalMatchGlob(key, pattern, matchOpts)
+  if (originalMatchGlob(key, pattern, matchOpts)) return true
 
-  if (result) return true
+  if (isRegex(pattern)) return regexMatch(key, asRegex(pattern))
+
+  if (!pattern.includes('*') && pattern.endsWith('/') && !warnedPatterns.has(pattern)) {
+    warnedPatterns.add(pattern)
+    renderWarning({
+      headline: 'Directory pattern may be misleading.',
+      body: `For more reliable matching, consider using ${pattern}* or ${pattern}*.filename instead.`,
```

Comment:
Users may have a more deterministic behavior when using strings, so I believe we could encourage them in that direction. Also, I think we mean `<file_extension>` since the `*` would represent the file name. Am I missing something?

```suggestion
      body: `For more reliable matching, consider using "${pattern}*" or "${pattern}*.<file_extension>" instead.`,
```

Reply:
You are absolutely right. That should have been referencing `file_extension`. Thank you!